### PR TITLE
Add caver-js via single package.json w/ npm auto-update

### PR DESCRIPTION
**Pull request by jimni1222**
# Pull request template for new library

Pull request for issue: #13431
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/klaytn/caver-js
* Official website (optional, not the repository): https://www.klaytn.com
* NPM package url (optional): https://www.npmjs.com/package/caver-js
* License and its reference: LGPL
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 6
  * Count of stars: 17
  * Count of forks: 25
* NPM download stats (optional):
  * Downloads in the last day: 393
  * Downloads in the last week: 459
  * Downloads in the last month: 2,275

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by jimni1222** [comment]
@zackbloom @MattIPv4 
Hello :) I have a question for you!

It's been a while since this PR was uploaded, and I didn't receive any feedback.
So I'm curious about what's going on that's why I leave a comment.

Please let me know if there are any omissions in this PR or if there are any problems.

Thank you. And Happy New Year :)

**Comment by MattIPv4** [comment]
```
ls node_modules/caver-js

CONTRIBUTING.md    LICENSE            TERMS-OF-USE.md    caver-cli          gulpfile.js        package.json       release_rc.sh      test
Changelog.md       README.md          breakingChanges.md cli.js             index.js           packages           rpc.json
```

I don't see the `dist` directory that you have referenced in the auto-update config?

File: ajax/libs/caver-js/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "license": "LGPL",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/klaytn/caver-js.git"
+  },
+  "homepage": "https://github.com/klaytn/caver-js",
+  "npmName": "caver-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```
Comment (MattIPv4) [inline ajax/libs/caver-js/package.json]: `dist` directory is not present in npm package.

Comment (jimni1222) [reply ajax/libs/caver-js/package.json]: Sorry, but the changes to add the dist folder to the npm package are now only released in the rc version.
I will request a review again after it is released in latest version!

File: ajax/libs/caver-js/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "files": [
+        "**/*"
```
Comment (xtuc) [inline ajax/libs/caver-js/package.json]:
```suggestion
        "*"
```

there are no subfolder in dist

File: ajax/libs/caver-js/package.json (hunk 1 above)
Comment (xtuc) [reply ajax/libs/caver-js/package.json]: the bot will publish rc so it should be fine

Comment (jimni1222) [reply ajax/libs/caver-js/package.json]: @xtuc 
Are you saying it will be on cdn for the rc version? 
I didn't know that! Thank you for letting me know :)

Comment (xtuc) [reply ajax/libs/caver-js/package.json]: Yes

**Comment by jimni1222** [comment]
@xtuc @MattIPv4 
Hello, since caver-js v1.3.2, which is distributed on npm, files are included in dist folder. 
If you have time, please review it once more and thank you :)

**Comment by jimni1222** [comment]
@xtuc @MattIPv4 
Hello. Thank you so much for this PR review. :) 
I am so happy to merge into the master branch.

How long does it take to be able to search on cdnjs.com after the branch merges?

**Comment by xtuc** [comment]
> How long does it take to be able to search on cdnjs.com after the branch merges?

It's fast but it failed for caver-js
```
ajax/libs/caver-js/package.json: version is invalid
```

I will switch the package to the new bot which sets the version

**Comment by jimni1222** [comment]
> > How long does it take to be able to search on cdnjs.com after the branch merges?
> 
> It's fast but it failed for caver-js
> 
> ```
> ajax/libs/caver-js/package.json: version is invalid
> ```
> 
> I will switch the package to the new bot which sets the version

Sounds good. :) 
Please tag me when you update the new bot !
Please let me know if you need anything from me :)

**Comment by jimni1222** [comment]
@xtuc 
Hi :) caver-js is not yet searchable on cdnjs.com.
Can I ask about the progress or schedule?

**Comment by jimni1222** [comment]
@xtuc @MattIPv4 
Hello :) !
When will I be able to use caver-js through cdnjs.com?
Feel free to let me know if there is something i can help with.

**Comment by xtuc** [comment]
I moved `caver-js` to the new bot, it should be on the website by tomorrow

**Pull request merged**